src/main/kotlin/dev/ocpd/jsensible/rules/spring/SpringRules.kt

```diff
@@ -32,4 +35,18 @@ object SpringRules {
     fun noJakartaTransactionOnMethod() =
         noMethods().should().beAnnotatedWith("jakarta.transaction.Transactional")
             .because("in Spring application, it is recommended to use [org.springframework.transaction.annotation.Transactional] instead")
+
+    /**
+     * Spring Data Repositories only support their own [org.springframework.lang.Nullable] annotation.
+     *
+     * Spring Data Repositories only support the usage of their own [org.springframework.lang.Nullable] annotation.
+     * Misuses could lead to runtime errors when invoking repository methods.
+     *
+     * Solution: Use the [org.springframework.lang.Nullable] annotation.
+     */
+    fun onlyUseSpringNullableAnnotation(): ArchRule =
+        noClasses()
+            .that().areAssignableTo("org.springframework.data.repository.Repository")
+            .should(notUseSpringNullable())
+            .because("in Spring Data Repositories, it only support the [org.springframework.lang.Nullable] annotation")
```

Comment:
align this because clause to the summary in the KDoc

Reply:
I don't understand what it means.